pkg/vpcmodel/responsive.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package vpcmodel
+
+import (
+	"github.com/np-guard/models/pkg/connection"
+	"github.com/np-guard/models/pkg/netp"
+)
+
+func newTCPSet() *connection.Set {
+	return connection.TCPorUDPConnection(netp.ProtocolStringTCP, connection.MinPort, connection.MaxPort,
+		connection.MinPort, connection.MaxPort)
+}
+
+// PartitionTCPNonTCP given a connection returns its TCP and non-TCP sub-connections
+func partitionTCPNonTCP(conn *connection.Set) (tcp, nonTCP *connection.Set) {
```

Comment:
how are these functions related to the file name `responsive.go`? these are more general functions to be placed elsewhere?

Reply:
If we move these two functions to, say, `commonConnectivity.go`, then responsive.go is left with a single function. Perhaps move all to `commonConnectivity.go`?

Reply:
Moved to `commonConnectivity.go`, empty `responsive.go` deleted

Reply:
ok, should `commonConnectivity.go` be moved to another pkg, rather than stay in `vpcmodel` ?
maybe open a separate issue to reorg entire pkg?

Reply:
sure. The package is too big. https://github.com/np-guard/vpc-network-config-analyzer/issues/638